working-groups/query-standardization/dsl-survey/ppl/README.md

```diff
@@ -0,0 +1,256 @@
+
+    * UTF-8.
+
+1. **Special or Reserved Characters**:
+
+    * Special characters like parentheses, commas, etc., have specific uses in PPL.
+
+1. **Writing Data or Configuring the Backing Store**:
+
+    * PPL is for querying only.
+
+* * *
+
+### **Metrics**
+
+* **Supported Metric Measurement Types**:
+    * Gauges, rates, counters, etc. PPL can work with different metric types both in OpenSearch and in Prometheus .
+* **Handling Changes to Measurement Type Over Time**:
+    * Changes in measurement type are handled at the data level; PPL queries the data as-is without managing type changes.
+* **Supported Values as Metrics**:
+    *  Double precision floating-point, integers, booleans, summary statistics, histogram bucket counts, etc.
+* **Temporal Aggregation**:
+    *  Supports custom aggregation with functions like
+        * [COUNT](https://github.com/opensearch-project/sql/blob/main/docs/user/ppl/cmd/stats.rst#count)
+        * [SUM](https://github.com/opensearch-project/sql/blob/main/docs/user/ppl/cmd/stats.rst#sum)
+        * [AVG](https://github.com/opensearch-project/sql/blob/main/docs/user/ppl/cmd/stats.rst#avg)
+        * [MAX](https://github.com/opensearch-project/sql/blob/main/docs/user/ppl/cmd/stats.rst#max)
+        * [MIN](https://github.com/opensearch-project/sql/blob/main/docs/user/ppl/cmd/stats.rst#min)
+        * [VAR_SAMP](https://github.com/opensearch-project/sql/blob/main/docs/user/ppl/cmd/stats.rst#var-samp)
+        * [VAR_POP](https://github.com/opensearch-project/sql/blob/main/docs/user/ppl/cmd/stats.rst#var-pop)
+        * [STDDEV_SAMP](https://github.com/opensearch-project/sql/blob/main/docs/user/ppl/cmd/stats.rst#stddev-samp)
+        * [STDDEV_POP](https://github.com/opensearch-project/sql/blob/main/docs/user/ppl/cmd/stats.rst#stddev-pop)
+        * [TAKE](https://github.com/opensearch-project/sql/blob/main/docs/user/ppl/cmd/stats.rst#take)
+* **Timestamp Alignment**:
+    * Typically aligned to the start of the window.
+* **Boundaries**:
+    *  Inclusive of the start, exclusive of the end.
+* **Aggregation Functions for Downsampling**:
+    * N/A - can use OpenSearch specific engine downsampling via [Index roolup](https://opensearch.org/docs/latest/im-plugin/index-rollups/index/)
+* **Determining Correct Aggregation Type if Fixed**:
+    * Not applicable as it's not fixed.
+* **Querying Over Pre-Computed Temporal Aggregates**:
+    * Both manual and automatic selection is supported, depending on the query and time range.
+
+**Interpolation**
+
+* **Interpolation Scheme**:
+    * Depends on the OpenSearch settings and query configuration (schema on read).
+* **User Selection of Interpolation Scheme**:
+    *  Not typically user-configurable at query time.
+* **Handling Missing Data**:
+    * Represented internally as null, NaN, etc., depending on the context.
+* **Handling Missing Data for Aggregations**:
+    * Specific behavior depends on the aggregation function and data type.
```

Comment:
Could you elaborate on the different contexts please? An example or two, doesn't have to be exhaustive.